sklearn/feature_extraction/text.py

```diff
@@ -1008,6 +1026,8 @@ class TfidfTransformer(BaseEstimator, TransformerMixin):
     Normalization is "c" (cosine) when ``norm='l2'``, "n" (none)
     when ``norm=None``.
 
+    By default, the l2 norm is used.
```

Comment:
I couldn't decide whether or not to go down a rabbit hole of l2 vs. l1 norm, partly because my first pass at a google search didn't reveal the most cogent explanations; feels out of scope for a docstring, though?